### ISSUE TITLE
Jetpack Revue: form is outputting broken html in the submit button wrapper div

### ISSUE THREAD

**Issue by gabrielamelo** [Automattic/jetpack#20924]
### Impacted plugin

Jetpack

### Steps to Reproduce

1. Add and set up Revue block widget

### What actually happened

Hello,
I am not sure if this is the right place to ask, but I have a situation with the Revue block: it is outputting a broken HTML in the submit button wrapper div.
This HTML does not make sense to me as it has a weird quote mark in it, but it seems to work fine with some WordPress themes, like Twenty Twenty-One, but not with mine.

The HTML in question (line 85 of `jetpack/extensions/blocks/button/button.php` in the plugin):

![Desktop-screenshot (4)](https://user-images.githubusercontent.com/24965103/131750697-2a27a1a8-3ffc-4524-add1-03c755eff03a.png)

How is showing on my theme: 

![Desktop-screenshot (3)](https://user-images.githubusercontent.com/24965103/131750735-07e488fd-687a-4f95-b720-f94852951b14.png)

So, this is something caused by the theme I developed? What should I look for in order to fix it? I did try to disability several things, widgets, plugins, and nothing seems to change this issue.

### Operating System

Linux

### Browser

Chrome/Chromium

**Comment by jeherve** [comment]
Good catch, thanks for the report. We'll get this fixed, it's definitely an issue on our end.

**Comment by gabrielamelo** [comment]
Thanks (:
